target/linux/kirkwood/base-files/etc/init.d/hwmon_fancontrol

```diff
@@ -7,6 +7,13 @@ boot() {
 	# for more information, please read https://www.kernel.org/doc/Documentation/hwmon/sysfs-interface
 
 	case $(board_name) in
+	ctera,c200-v1)
+		path_to_hwmon='/sys/devices/platform/ocp@f1000000/f1011000.i2c/i2c-0/0-004c/hwmon/hwmon0'
+
+		# It should be related to hdd temerature instead lm63 temp
```

Comment:
Does this device have a fan?

Reply:
Yes.

Reply:
Ah, now I think I found out what setting 1 to `pwm1_enable` means, so these lines detach the fan control from lm63 temp and set the fan speed to a constant, right?

Reply:
Exactly.

The fan is blowing on HDD bays, not on lm63. IMO, the best approach is to get temp from  SMART and use it for fan control. But it can't be done in core image.